src/buildercore/bootstrap.py

```diff
@@ -489,7 +450,7 @@ def update_stack(stackname, service_list=None, **kwargs):
         ('ec2', update_ec2_stack),
         ('s3', update_s3_stack),
         ('sqs', update_sqs_stack),
-        ('terraform', update_terraform_stack)
+        ('terraform', lambda stackname, context: terraform.update(stackname, context))
```

Comment:
this is redundant and can be rewritten like: `('terraform', terraform.update),`

Reply:
nice that this isn't JavaScript :muscle: